src/main/java/net/mcreator/ui/laf/MCreatorLookAndFeel.java

```diff
@@ -27,18 +28,21 @@
 import javax.swing.text.html.HTMLEditorKit;
 import javax.swing.text.html.StyleSheet;
 import java.lang.reflect.Field;
+import java.util.Objects;
 
 public class MCreatorLookAndFeel extends MetalLookAndFeel {
 
 	private static final Logger LOG = LogManager.getLogger("Look and Feel");
 
-	private final AbstractMCreatorTheme theme;
+	private MCreatorTheme theme;
 
 	public MCreatorLookAndFeel() {
-		if (PreferencesManager.PREFERENCES.ui.interfaceTheme.equals("Light theme")) {
-			setCurrentTheme(theme = new LightMCreatorTheme());
-		} else {
-			setCurrentTheme(theme = new DarkMCreatorTheme());
+		try {

```

Comment:
Do null check, do not just catch exception and wrap with requireNonNull, also log with WARN when default theme is used due to missing theme.